tools/actions/generator.go

```diff
@@ -119,6 +123,28 @@ func (g Generator) createActionTestFile() error {
 	return nil
 }
 
+func (g Generator) createActionTemplate(root string) error {
```

Comment:
@YanDeLeon as this is also done by other generator we should find a way to make this generator call the other one, to me that should be an interface that the Template generator meets, so this plugin can invoke those that are needed.

Reply:
Wich generator is the one that do it ?